css/styles.css

```diff
@@ -80,3 +86,64 @@ input {
 .text-right {
   text-align: right;
 }
+
+.row-entries-header {
+  display: flex;
+}
```

Comment:
Why is this class necessary? You should just reuse the `row` class you made above